internal/services/delegation.go

```diff
@@ -221,14 +220,19 @@ func (s *Service) processBTCDelegationInclusionProofReceivedEvent(
 		}
 	}
 
-	// Update delegation details
-	if dbErr := s.db.UpdateBTCDelegationDetails(
+	stakingStartHeight, _ := utils.ParseUint32(inclusionProofEvent.StartHeight)
+	stakingEndHeight, _ := utils.ParseUint32(inclusionProofEvent.EndHeight)
```

Comment:
so I guess that `inclusionProofEvent` fields will always contain correct uint32 strings, can you point out where this check / conversion to string implemented? Also if it's always `uint32` inside a string, why not to change `inclusionProofEvent.*Height` from string to uint32 ?